api/src/main/java/org/pmiops/workbench/jira/JiraService.java

```diff
@@ -0,0 +1,163 @@
+package org.pmiops.workbench.jira;
+
+import com.google.common.collect.ImmutableMap;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.inject.Provider;
+import org.pmiops.workbench.jira.api.JiraApi;
+import org.pmiops.workbench.jira.model.AtlassianContent;
+import org.pmiops.workbench.jira.model.AtlassianDocument;
+import org.pmiops.workbench.jira.model.Comment;
+import org.pmiops.workbench.jira.model.CreatedIssue;
+import org.pmiops.workbench.jira.model.IssueTypeDetails;
+import org.pmiops.workbench.jira.model.IssueUpdateDetails;
+import org.pmiops.workbench.jira.model.Project;
+import org.pmiops.workbench.jira.model.SearchRequestBean;
+import org.pmiops.workbench.jira.model.SearchResults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service to mediate interactions with the RW Jira project.
+ *
+ * <p>To get the schema for RW issues (including all field names):
+ *
+ * <p><code>
```

Comment:
nit, you can use code block (`<pre><code>`):
go/java-practices/javadoc#formatting_excerpts

Reply:
done